lib/web/scripts/desktop/configure-ad.ps1

```diff
@@ -0,0 +1,170 @@
+$TELEPORT_CA_CERT_PEM = "{{.caCertPEM}}"
```

Comment:
Why does this script not have `$ErrorActionPreference = "Stop"`??

Reply:
There is no reason it shouldn't have it. Added that line to the script.